Greenstein-ebay-list-item

m-greenstein opened the pull request:
This is my PR for my first scenario. Please feel free to review at your convenience, as I know this is late.

evreddic commented on youngm/young-pavlova.md:
```diff
@@ -19,7 +21,13 @@ A meringue may seem intimidating but it is just a combination of egg whites and
+
 6. Add in the vanilla extract. Whip for one more minute. 
 7. Finally, add in the cream of tartar and cornstarch. [^4]
 8. Spread the Pavlova out onto the pan, you can get creative with the shaping and sizing. But the one consistent detail is a dip in the center and relatively tall edges.
```
Hey, 

It seems you have already merged your changes but just wanted to give some quick feedback for your procedure. I believe your overall procedure has a nice transition from staging to coaching just like a recipe procedure should. I think in the coaching section, having links separating the steps in the procedure could be a little distracting just because this appears to be a recipe where your constantly moving or requires constant attention. Maybe have the links before the coaching stage so they could study up instead of having to stop in the middle of a step. I think this strategy could work in other types of procedures though.

m-greenstein committed:
Copied + renamed TOCOPY folder

Renamed main folder and dded three procedure markdown files

m-greenstein committed:
Created headings + started formatting procedure

Divided the steps into three sections and began formatting the steps inside the sections

m-greenstein committed:
Add section 1 step 2

m-greenstein committed:
Added section 1 steps 3 and 4

m-greenstein committed:
Copied the rest of section 2

Copied section 2 from a notes document; still updating formatting and proofreading, but the content is there

m-greenstein committed:
Finished formatting section 2

m-greenstein committed:
*Actually* finished formatting section 2

m-greenstein committed:
Update greenstein-ebay-list-item.md

m-greenstein committed:
Added an alerting move

lingeringcode committed:
Create ebay-register.png

m-greenstein committed:
Update greenstein-ebay-list-item.md

m-greenstein committed:
Added images

Added 7 images throughout the procedure

m-greenstein committed:
Uploaded user scenario

lingeringcode committed:
Feedback from Lindgren

m-greenstein committed:
Update parts 1/2 of "list item"

m-greenstein committed:
Update phases 3 and 4 of "list item"
